src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from "@chakra-ui/react"
+import { Story, ComponentMeta } from "@storybook/react"
+
+import { Footer } from "./Footer"
+
+const footerMeta = {
+  title: "Components/Footer",
+  component: Footer,
+} as ComponentMeta<typeof Footer>
+
+interface TemplateArgs {
+  buttonText: string
+}
+
+const SingleButtonFooterTemplate: Story<TemplateArgs> = ({
+  buttonText,
+}: TemplateArgs) => {
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <Footer>
+      <Button>{buttonText}</Button>
+    </Footer>
+  )
+}
+
+interface MultiButtonFooterTemplateArgs extends TemplateArgs {
+  button2Text: string
```

Comment:
could we change the name to avoid mixing numbers/letters in names

Reply:
Changed in https://github.com/isomerpages/isomercms-frontend/pull/932/commits/88f6867043db46c09fcb43abfad99a66a315a766!